website/source/guides/acl.html.markdown

```diff
@@ -0,0 +1,310 @@
+  configuration, it may take only a single server failure for cluster
+  unavailability. Recovery requires an operator to intervene, but recovery is
+  straightforward.
+---
+
+# ACL System
+
+Nomad provides an optional Access Control List (ACL) system which can be used to control access to data and APIs. The ACL is [Capability-based](https://en.wikipedia.org/wiki/Capability-based_security), relying on tokens which are associated with policies to determine which fine grained rules can be applied. It is very similar to [AWS IAM](https://aws.amazon.com/iam/) in many ways.
+
+# ACL System Overview
+
+The ACL system is designed to be easy to use, fast to enforce, and flexible to new policies, all while providing administrative insight. At the highest level, there are three major components to the ACL system:
+
+![ACL Overview](/assets/images/acl.jpg)
+
+ * **ACL Policies**. No permissions are granted by default, making Nomad a default-deny or whitelist system. Policies allow a set of capabilities or actions to be granted or whitelisted. For example, a "readonly" policy might only grant the ability to list and inspect running jobs, but not to submit new ones.
+
+ * **ACL Tokens**. Requests to Nomad are authenticated by using bearer token. Each ACL token has a public Accessor ID which is used to name a token, and a Secret ID which is used to make requests to Nomad. The Secret ID is provided using a request header (`X-Nomad-Token`) and is used to authenticate the caller. Token are either `management` or `client` types. The `management` tokens are effectively "root" in the system, and can perform any operation. The `client` tokens are associated with one or more ACL policies which grant specific capabilities.
+
+ * **Capabilities**. Capabilties are the set of actions that can be performed. This includes listing jobs, submitting jobs, querying nodes, etc. A `management` token is granted all capabilities, while `client` tokens are granted specific capabilties via ACL Policies. The full set of capabilities is discussed below in the rule specifications.
+
+### ACL Policies
+
+An ACL policy is a named set of rules. Each policy must have a unique name, an optional description, and a rule set.
+A client ACL token can be associated with multiple policies, and a request is allowed if _any_ of the associated policies grant the capability.
+Management tokens cannot be associated with policies because they are granted all capabilities.
+
+The special `anonymous` policy can be defined to grant capabilities to requests which are made anonymously. If a request is made to Nomad without the `X-Nomad-Token` header specified, then it is an anonymous request. This can be used to allow anonymous users to list jobs and view their status, while requiring authenticated requests to submit new jobs or modify existing jobs. By default, there is no `anonymous` policy set meaning all anonymous requests are denied.
```

Comment:
An anonymous request is a request made to Nomad without the `X-Nomad-Token` header specified.